<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentRegistration</class>
 <widget class="QMainWindow" name="studentRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>650</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>431</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>welcome_screen logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>80</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Agency FB&quot;;
font-weight: 900;
color: rgb(75, 103, 118);
background-color: none;</string>
     </property>
     <property name="text">
      <string>REGISTR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Idno_lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 103, 118);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fname_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 103, 118);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lname_lineEdit_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 103, 118);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_lineEdit_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>445</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 103, 118);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_lineEdit_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>540</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 103, 118);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Idno">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>65</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(75, 103, 118);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>ID No:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fname">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13.5pt &quot;Georgia&quot;;
color: rgb(75, 103, 118);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>First Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lname">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>111</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(75, 103, 118);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Last Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="address">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(75, 103, 118);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Address :</string>
     </property>
    </widget>
    <widget class="QLabel" name="phone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>535</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(75, 103, 118);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Phone No.:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Agency FB&quot;;
font-weight: 900;
color: rgb(75, 103, 118);
background-color: none;</string>
     </property>
     <property name="text">
      <string>STUD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>590</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Rockwell&quot;;
color:rgb(0,0,0);
border-radius: 15px;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(73, 135, 255, 255), stop:1 rgba(185, 91, 225, 255));</string>
     </property>
     <property name="text">
      <string>ADD STUDENT</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>475</width>
      <height>650</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(75, 103, 118);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-240</x>
       <y>170</y>
       <width>451</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>welcome_screen logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>80</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Agency FB&quot;;
font-weight: 900;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>RATION</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_lineEdit_11">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>540</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
border: none;
border-radius: 5px;
font: 12pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_lineEdit_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>445</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
border: none;
border-radius: 5px;
font: 12pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Gender_lineEdit_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
border: none;
border-radius: 5px;
font: 12pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Email_lineEdit_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
border: none;
border-radius: 5px;
font: 12pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="course_lineEdit_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
border: none;
border-radius: 5px;
font: 12pt &quot;Verdana&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Gender">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>150</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(255,255,255);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Gender :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Email">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>250</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(255,255,255);
font-weight: 500;
background-color: none;</string>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QLabel" name="course">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>350</y>
       <width>131</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(255,255,255);
font-weight: 500;
background-color: none;
</string>
     </property>
     <property name="text">
      <string>Course :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Username">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>440</y>
       <width>131</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(255,255,255);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>540</y>
       <width>131</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Georgia&quot;;
color: rgb(255,255,255);
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Agency FB&quot;;
font-weight: 900;
color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>DENT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Rockwell&quot;;
color:rgb(0,0,0);
border-radius: 15px;
background-color: qlineargradient(spread:pad, x1:0.329727, y1:0.432, x2:0.938, y2:0.619318, stop:0.215909 rgba(162, 223, 176, 255), stop:0.698864 rgba(200, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
